app/data_providers/snowplow/config.py

```diff
@@ -20,14 +20,6 @@ class SnowplowConfig:
     METHOD: Method = 'post'
     BUFFER_SIZE: int = 1  # Number of events to buffer before flushing to Snowplow. 1 = No buffer.
 
-    CORPUS_SLATE_SCHEMA = 'iglu:com.pocket/corpus_slate/jsonschema/3-0-0'
-    CORPUS_SLATE_LINEUP_SCHEMA = 'iglu:com.pocket/corpus_slate_lineup/jsonschema/2-0-0'
-    USER_SCHEMA = 'iglu:com.pocket/user/jsonschema/1-0-0'
-    OBJECT_UPDATE_SCHEMA = 'iglu:com.pocket/object_update/jsonschema/1-0-8'
-    FEATURE_FLAG_SCHEMA = 'iglu:com.pocket/feature_flag/jsonschema/1-0-0'
-    VARIANT_ENROLL_SCHEMA = 'iglu:com.pocket/variant_enroll/jsonschema/1-0-0'
-    API_USER_SCHEMA = 'iglu:com.pocket/api_user/jsonschema/1-0-1'
```

Comment:
I moved these to `entities.py` because the Snowplow schema versions are directly tied to the code that constructs the entities. I doesn't make sense to me to put these in a config where they have to be passed around.